shell/browser/ui/inspectable_web_contents_impl.cc

```diff
@@ -805,22 +805,6 @@ bool InspectableWebContentsImpl::DidAddMessageToConsole(
   return true;
 }
 
-bool InspectableWebContentsImpl::ShouldCreateWebContents(
-    content::WebContents* web_contents,
-    content::RenderFrameHost* opener,
-    content::SiteInstance* source_site_instance,
-    int32_t route_id,
-    int32_t main_frame_route_id,
-    int32_t main_frame_widget_route_id,
-    content::mojom::WindowContainerType window_container_type,
-    const GURL& opener_url,
-    const std::string& frame_name,
-    const GURL& target_url,
-    const std::string& partition_id,
-    content::SessionStorageNamespace* session_storage_namespace) {
-  return false;
```

Comment:
was this doing something useful...?

Reply:
We were overriding the default value, now the default value with the new method is what we prefer https://chromium-review.googlesource.com/c/chromium/src/+/1846477.